recipes/neon2sse/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class neon2sseConan(ConanFile):
+    name = "neon2sse"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/intel/ARM_NEON_2_x86_SSE"
+    description = "Header only library intended to simplify ARM->IA32 porting"
+    license = "BSD-2-Clause"
+    topics = "neon", "sse", "port", "translation", "intrinsics"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+    
+    def validate(self):
+        if "x86" not in self.settings.arch:
```

Comment:
This adds "type safety" :)
```suggestion
        if self.settings.arch not in ("x86", "x86_64"):
```

Reply:
Hmm 
What could go wrong with the way I did it?

Reply:
The behavior is the same when no other arch contains the string "x86".
But when using "my" syntax, conan will check whether the arch string that is tested again is also a valid arch.

Basically, you're doing a string comparison, whereas I do arch comparison.

Reply:
Yup I wanted to catch both x86 and x86_64. But let's keep it explicit.